File: Selenium/Activities/Activity_22.py
from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/popups")
    print("The page title is: ", driver.title)
    driver.maximize_window()
    msg = driver.find_element(By.XPATH, "//button[@onclick='openModal()']").text
    print("The Signup Page message is: ",msg)
    driver.find_element(By.XPATH, "//button[@onclick='openModal()']").click()
    driver.find_element(By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "password").send_keys("password")
    driver.find_element(By.XPATH, "//button[@onclick='signIn()']").click()
    print("log in confirmation message is ", driver.find_element(By.ID, "action-confirmation").text)
    driver.quit()

File: Selenium/Activities/Activity_22.java
package Activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_22 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/popups");
        System.out.println("The Page title is: " + driver.getTitle());
        String msg = driver.findElement(By.xpath("//button[@onclick='openModal()']")).getText();
        System.out.println("The Signup Page message is: " + msg);
        driver.findElement(By.xpath("//button[@onclick='openModal()']")).click();
        driver.findElement(By.id("username")).sendKeys("admin");
        driver.findElement(By.id("password")).sendKeys("password");
        driver.findElement(By.xpath("//button[@onclick='signIn()']")).click();
        String Conf_msg = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println(Conf_msg);
        driver.quit();

    }
}
